src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -295,6 +295,58 @@ void SubgraphBaseTest::configure_model() {
     function = p.build();
 }
 
+bool SubgraphBaseTest::is_allowed_convertion(ov::element::Type from, ov::element::Type to) {
+// based on logic from /src/common/transformations/src/transformations/convert_precision.cpp
+// and some restrictions on converting real to integral and vice versa from ops
+    if (from == to) {
+        return false;
+    }
+    if (from.is_real()) {
+        if ((from == ov::element::Type_t::f32 && to == ov::element::Type_t::f16) || to == ov::element::Type_t::f32) {
+            return true;
+        }
+    } else if (from.is_integral() && from.is_signed()) {
+        if ((from == ov::element::i4) ||
```

Comment:
i16?

Reply:
according to transformations/convert_precision.cpp i4, but I removed it at all now, to avoid situation when integral convert to real